draft/2020-06-02-this-week-in-rust.md

```diff
@@ -35,6 +34,18 @@ Check out [this week's *This Week in Rust Podcast*]()
 * [Why I'm enjoying learning Rust as a Java programmer](https://opensource.com/article/20/5/rust-java)
 * [Contributing to Rust](https://blog.elinvynia.com/posts/2020-05-26-contributing-to-rust.html)
 * [Get a Look on Key Rust Crates for WebAssembly](https://blog.knoldus.com/get-a-look-on-key-rust-crates-for-webassembly/)
+* [10 Most Loved Programming Languages: Rust, TypeScript, and More](https://insights.dice.com/2020/05/29/10-most-loved-programming-languages-rust-typescript-more/)
+* [Compiling Rust binaries for Windows 98 SE and more: a journey](https://seri.tools/blog/compiling-rust-for-legacy-windows)
```

Comment:
Looks like we had this link in [last week's issue](https://github.com/emberian/this-week-in-rust/blob/master/content/2020-05-27-this-week-in-rust.md#news--blog-posts)